src/main/java/io/github/debuggyteam/architecture_extensions/resource/DataGeneration.java

```diff
@@ -25,7 +25,7 @@
 import net.minecraft.util.Identifier;
 
 public final class DataGeneration {
-	static final Set<BlockType.TypedGroupedBlock> BLOCKS = Sets.newHashSet();
+	public static final Set<BlockType.TypedGroupedBlock> BLOCKS = Sets.newHashSet();
```

Comment:
Burying the lead here. Is this a thing we want available for API?

Reply:
No, but it's needed in the client class